spec/que/job.bulk_enqueue_spec.rb

```diff
@@ -44,6 +44,7 @@ def assert_enqueue(
       assert_equal expected_job_class.to_s, job[:job_class]
       assert_equal expected_args[i], job[:args]
       assert_equal expected_kwargs[i], job[:kwargs]
+      assert_equal job[:run_at], job[:first_run_at] if Que.db_version >= 8
```

Comment:
Why have you added all these db schema version condition checks? The specs only need to run against the latest db schema (other than the specs that test the migrations themselves)